patches/minecraft/net/minecraft/SharedConstants.java.patch

```diff
@@ -1,5 +1,13 @@
 --- a/net/minecraft/SharedConstants.java
 +++ b/net/minecraft/SharedConstants.java
+@@ -102,6 +_,7 @@
+    public static final long f_136181_ = Duration.ofMillis(300L).toNanos();
+    public static boolean f_136182_ = true;
+    public static boolean f_136183_;
++   public static final java.util.function.BooleanSupplier ENABLE_GAMETEST = () -> f_136183_ || Boolean.getBoolean("forge.enablegametest");
```

Comment:
Just out of total curiosity, is there any reason this needs to be a patch at all? It could totally be located in forge code somewhere. Like, yes, vanilla has all it's constants here but since forge is adding stuff, there's no reason forge needs to put it's constants here as well.

You could also then make it a method rather than this odd "field which is a boolean supplier yet only used as a method" i.e. `ForgeMod.areGameTestsEnabled();`